app-backend/database/src/main/scala/com/azavea/rf/database/Database.scala

```diff
@@ -40,7 +40,13 @@ class Database(jdbcUrl: String, dbUser: String, dbPassword: String, slickThreadC
 
   val db = {
     val executor = AsyncExecutor("slick", numThreads = slickThreadCount, queueSize = slickQueueSize)
-    driver.api.Database.forDataSource(dataSource, executor)
+
+    driver.api.Database.forDataSource(
+      ds = dataSource,
+      maxConnections = None,
+      executor = executor,
+      keepAliveConnection = false
+    )
```

Comment:
The latest GT bumped a version of a Postgres plugin, which for some reason caused RF compiles to break. These lines here are part of the fix.